timescaledb/how-to-guides/update-timescaledb/downgrade-timescaledb.md

```diff
@@ -0,0 +1,29 @@
+# Downgrade TimescaleDB versions
+If you upgrade your TimescaleDB version and encounter problems, you can
+immediately roll back your upgrade. To downgrade back to your previous version,
+connect to your database and run:
+
+```sql
+ALTER EXTENSION timescaledb UPDATE TO '<PREVIOUS_VERSION>';
+```
+
+For example:
+```sql
+ALTER EXTENSION timescaledb UPDATE TO '2.5.1';
+```
+
+<highlight type="important"> 
+If you're connecting to your database with `psql`,
+connect with the `-X` flag. This prevents any commands in `.psqlrc` from loading
+another TimescaleDB version. 
+</highlight>
+
+You can verify that the downgrade worked by running `\dx timescaledb` to see the
+extension's version number.
+
+## Limitations
+
+The downgrade script is tested for a single-step downgrade from the current
+version to the previous version. Multi-step downgrades may work but have not
+been tested. Downgrading may not work if you make changes to your database
+between upgrading and downgrading.
```

Comment:
```suggestion
between upgrading and downgrading.

Downgrading is not guaranteed to be supported for all versions. We do our best to ensure that downgrading should work both between patch versions and also to a previous minor version, but it might not always be possible to support this. If you rely on being able to downgrade, please check the release notes to make sure that it supports downgrading before you do an upgrade.
```